SECURITY_POLICY.txt

```diff
@@ -0,0 +1,75 @@
+MapLibre Security and Vulnerability Reporting Policy
+
+1. Services Covered by this Policy
+
+This policy covers all services directly operated by us (MapLibre).
+Services can be identified by the following means:
+- The website has a .well-known/security.txt that links to this policy.
+- The reverse DNS of an IP address resolves to one of the following
+  domain (or subdomains): maplibre.org
+
+2. Acceptable Use
+
+We generally invite security researchers to search for vulnerabilities
+in our services. We kindly ask to not put any actual user data or
+production systems at risk.
+
+3. Classification of Vulnerabilities
+
+We will consider a vulnerability report most likely as relevant if it
+reports one of the following problems:
+- Memory-safety issues in any MapLibre project
+- The vulnerability can be used to directly access non-public
+  information that either reveals further security relevant problems or
+  contains user data, credentials, or sensitive data in general.
+- The vulnerability can be used to disrupt the orderly operation of a
+  service (Denial of Service).
+- The vulnerability can be used to manipulate data within the service.
+- XSS, CSRF, RCE, authentication/authorization bypass, SQL inections,
+  etc are considered relevant.
+
+We will consider a vulnerability report most likely as NOT relevant if
+it reports one of the following problems:
+- Missing security features, for example HTTP headers, if they are not
+  actually preventing a vulnerability.
+- Publicly accessible version strings of used software.
+- Security vulnerablities that can only be used within the scope of the
+  used account.
+
+4. Reporting Vulnerabilities
+
+Report vulnerabilities via e-mail to security@maplibre.org. For
+encryption the PGP key https://example.com/pubkey.asc can be used.
```

Comment:
I'm not sure if we want to allow encrypting emails. I could do the key management but in the end we probably should only use it if we somehow can ensure that the keys are usable by at least 2-3 people at MapLibre.

Reply:
yeah, PGP might be  an overkill at this stage